image/gen/src/generator.rs

```diff
@@ -109,6 +117,10 @@ impl<Crypto: ImageGeneratorCrypto> ImageGenerator<Crypto> {
                 &config.vendor_config.pub_keys.ecc_pub_keys[ecc_key_idx as usize],
             )?;
             vendor_sigs.ecc_sig = sig;
+            let lms_sig =
```

Comment:
With this PR, the time to run the tests on my workstation increases from 30 seconds to 234 seconds. This is unacceptable.

We need to reduce the number of tests that do a full LMS sign, or we need to make this signing operation more than an order of magnitude faster.